lib/taskana-core-test/src/test/java/acceptance/task/requestreview/RequestReviewAccTest.java

```diff
@@ -0,0 +1,131 @@
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+
+    defaultObjectReference = DefaultTestEntities.defaultTestObjectReference().build();
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_RequestReview_WhenTaskIsClaimed() throws Exception {
+    Task task = createTaskClaimedByUser_1_1().buildAndStore(taskService);
+    Instant now = Instant.now();
+    Task result = taskService.requestReview(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ForceRequestReview_WhenTaskIsClaimedByDifferentUser() throws Exception {
+    Task task = createDefaultTask().owner("user-1-2").buildAndStore(taskService);
+    Instant now = Instant.now();
+    Task result = taskService.forceRequestReview(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_RequestingReviewAndTaskIsClaimedByDifferentOwner()
+      throws Exception {
+    Task task =
+        createDefaultTask().owner("user-1-2").state(TaskState.CLAIMED).buildAndStore(taskService);
+    ThrowingCallable call = () -> taskService.requestReview(task.getId());
+
+    assertThatThrownBy(call)
+        .isInstanceOf(InvalidOwnerException.class)
+        .hasMessageContaining("User 'user-1-1' is not owner");
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_TaskIsNotClaimedYet() throws Exception {
+    Task task = createDefaultTask().buildAndStore(taskService);
+    ThrowingCallable call = () -> taskService.requestReview(task.getId());
+
+    assertThatThrownBy(call)
+        .isInstanceOf(InvalidTaskStateException.class)
+        .hasMessageContaining("[CLAIMED]");
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_ForceRequestReviewAndTaskIsInEndState() throws Exception {
+    Task task = createTaskClaimedByUser_1_1().buildAndStore(taskService);
+    taskService.completeTask(task.getId());
+    ThrowingCallable call = () -> taskService.forceRequestReview(task.getId());
+
+    assertThatThrownBy(call)
+        .isInstanceOf(InvalidTaskStateException.class)
+        .hasMessageContaining("[CLAIMED, READY]");
+  }
+
+  private TaskBuilder createTaskClaimedByUser_1_1() {
+    return createDefaultTask()
+        .owner("user-1-1")
+        .created(Instant.parse("2018-01-29T15:55:00Z"))
```

Comment:
technically you don't have to set the created timestamp :) 
It is more accurate if you do, but we don't use that timestamp anywhere in the tests ^^

Reply:
Ok :)